src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/dashboard.yaml

```
@@ -1,49 +1,49 @@
 sylius_twig_hooks:
     hooks:
-        'sylius_shop.account.dashboard.index.content':
+        'sylius_shop.account.dashboard.index.content.main':
             main_section:
-                template: '@SyliusShop/account/dashboard/index/content/main_section.html.twig'
+                template: '@SyliusShop/account/dashboard/index/content/main/main_section.html.twig'
```

And according to above:
```suggestion
                template: '@SyliusShop/account/dashboard/index/content/main/section.html.twig'
```